src/components/LoaderButtonComponent.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Button, Glyphicon } from 'react-bootstrap';
+import '../assets/loaderButton.css';
+
+export default ({
```

Comment:
If you're using the AirBnB style-guide, which the Andela style-guide extends, your javascript code should be indented with 2-spaces

Reply:
Noted